gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -239,6 +241,7 @@ def filter_subpop_qc(
         ld_r2=ld_r2,
         filter_decoy=False,
         checkpoint_path=get_checkpoint_path("intermediate_qc_mt", mt=True),
+        partitions=partitions,
```

Comment:
If you take suggestion to rename
```suggestion
        n_partitions=n_partitions,
```